### PR TITLE
fix: can't load directives with modifiers

### LINKED ISSUE #99: [Bug Report] v-intersect raises exception if used with modifiers
### Environment
**Vuetify Version:** 2.1.0
**Vue Version:** 2.6.10
**Browsers:** Chrome 78.0.3904.97
**OS:** Linux x86_64

### Steps to reproduce
Check out the repository, serve the app and open the console.

### Expected Behavior
The usage of the `v-intersect.quiet` directive in `src/App.vue` should not raise an exception.

### Actual Behavior
The directive raises `Failed to resolve directive: intersect` as if vue would not know the directive.
This also happens if one uses the `once` modifier.
BUT, everything works as expected if no modifier is used.

### Reproduction Link
<a href="https://github.com/Kritten/intersect-bug" target="_blank">https://github.com/Kritten/intersect-bug</a>

### Other comments
It is not reproduceable neither on codepen nor on codesandbox. 
Only in "real", local projects.

**Edit**
It works if one imports the directive directly in the component (many thanks to @jacekkarczmarczyk from discord ;))

### PR DESCRIPTION
**Pull request by jacekkarczmarczyk**
fixes #99

repo for testing: https://github.com/Kritten/intersect-bug

### CODE REVIEW

**Comment by KaelWD** [comment]
Couldn't you just use `node.directives` instead of `node.attrsList`?

File: lib/loader.js [hunk 1]
```diff
@@ -101,7 +101,9 @@ module.exports = async function (content, sourceMap) {
       compiler.compile(component.template.content, {
         modules: [{
           postTransformNode: node => {
-            node.attrsList.forEach(({ name }) => attrs.add(name))
+            if ("directives" in node) {
+              node.directives.forEach(({ name }) => attrs.add(name))
```
Comment (jacekkarczmarczyk) [inline lib/loader.js]: Hmm, technically it's a breaking change, if someone is using vuetify-loader to import something based on normal attributes it won't work anyway

We could keep the "attrs" mode as it was before (but without default matcher, still might be a breaking change but at least people will have possibility to do what they want with normal attrs) and add a directive matcher as a separate thing. Thoughts?

Comment (KaelWD) [reply lib/loader.js]: I don't think that's necessary. If someone requests it sure, but this option isn't documented and I doubt very many people are even using the one that is.

Comment (KaelWD) [reply lib/loader.js]: Also "something" would have to be a directive anyway because it's added to `component.options.directives`

**Pull request merged**